rlbot_gui/gui/js/story-mode-start.js

```diff
@@ -0,0 +1,89 @@
+
+
+export default {
+    name: 'story-start',
+    template: `
+    <b-container class="pt-5">
+    <b-jumbotron header="Story Mode" lead="Go on a Rocket League adventure">
+    </b-jumbotron>
+    <b-card>
+    <b-card-text>
+    <b-form @submit.prev="$emit('started', form)">
+        <b-form-group label="Teamname" label-for="teamname_entry" label-cols="auto">
+            <b-form-input 
+                type="text"
+                required
+                placeholder="Enter team name"
+                id="teamname_entry" v-model="form.teamname">
+            </b-form-input>
+
+        </b-form-group>
+
+        <b-form-group label="Team Color" label-cols="auto" v-if="colors">
+        <b-dropdown text="Pick color" id="team-color-picker">
```

Comment:
Picking a team color doesn't actually do anything for now, right?

Reply:
It doesn't and neither does name. Do you think its worth removing? Team customization is next big feature I want to add so I think I could use these then.

Reply:
Let's keep it, it feels exciting to customize something